### PR TITLE
Define Azure 11.2.1 WIP release

### PR DESCRIPTION
**Pull request by sslavic**
~This also deactivates 11.2.0~

~PR should be approved only once release notes are prepared and ready to share.~

Since @giantswarm/team-celestial is taking over as lead for Azure 11.2.1 release, this PR just defines a WIP without activating it or deactivating previous release.

### CODE REVIEW

**Comment by fiunchinho** [comment]
Updated `azure-operator` to it's latest release `v3.0.0` which includes k8s `v1.16.7`.

**Comment by rossf7** [comment]
@sslavic See https://github.com/giantswarm/releases/pull/150#issuecomment-597054409

I'd like to update to chart-operator 0.12.0 here. Is that OK?

**Comment by sslavic** [comment]
@rossf7 this release is part of a batch of releases to be jointly released, shipping by default enabled HPA for nginx including still to be released cluster-operator + nginx changes for cluster profiles
- AWS 9.0.1 (legacy, k8s 1.15, @tomahawk28 driving this part)
- AWS 9.2.1 (legacy, k8s 1.16)
- AWS 11.0.2 (node pools)
- KVM 11.2.1
- Azure 11.2.1

Releases already multiple changes, so I guess adding one more won't harm, as long as testing/release notes work can be completed today, so we can ship tomorrow.

Release notes WIP PR for all is maintained in https://github.com/giantswarm/releases/pull/141

Consider adding it to other releases in the batch. Most importantly, please help with release notes in that PR and test your changes with the WIP release definitions that you end up changing.

**Comment by rossf7** [comment]
@sslavic Ack. Thanks.  I think we should include it and happy to help with testing and release notes.

**Comment by rossf7** [comment]
I updated chart-operator to 0.12.1. Already tested on `ghost` with no issues. I'll do a final retest now.

**Comment by sslavic** [comment]
@giantswarm/team-celestial I plan to do one more round of testing of this WIP, focusing on our changes; if all OK, would you be fine with having this PR merged to master and you can take it from there, make any additional changes and finally activating it?

**Comment by MartaD** [comment]
@giantswarm/team-celestial Is it k8s 1.16.7 or 1.16.8?

**Comment by axbarsan** [comment]
> @giantswarm/team-celestial Is it k8s 1.16.7 or 1.16.8?

`azure-operator` master says `1.16.7`, so I believe it's correct
https://github.com/giantswarm/azure-operator/blob/c2576497329dc3c3d4d1158ccf2b7aba1cc32ce0/pkg/project/version_bundle.go#L38

**Comment by tuommaki** [comment]
> so I believe 1.16.7 is correct

With the latest tests and advancements, we are now targeting `1.16.8` which is the latest upstream patch release containing bunch of fixes for Azure VMSS API and disk management. These are crucial in order to avoid and in worst case live with the API rate-limiting.

**Comment by sslavic** [comment]
@giantswarm/team-celestial as you know unfortunately this WIP cannot be tested at the moment; I'm proposing to have this PR merged as-is to signal intent that this is worked on, and have all changes to master be in sync with it; we will iterate on the release (and its release notes) in follow up PRs, and of course testing before final activation; if you agree please ack and I'll merge it.

File: azure.yaml [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
+    - name: azure-operator
+      version: 3.0.0
+    - name: cert-operator
+      version: 0.1.0
+    - name: cluster-operator
+      provider: azure
+      version: 0.23.8
+  components:
+    - name: kubernetes
+      version: 1.16.7
```
Comment (tuommaki) [inline azure.yaml]:
```suggestion
      version: 1.16.8
```

**Pull request merged**